wherehows-web/app/templates/components/dataset-aclaccess.hbs

```diff
@@ -98,7 +99,7 @@
                 {{moment-format calendar.center 'MMMM YYYY'}}
 
                 <p>
-                  {{if hasValidExpiration (concat "Expires in " (moment-from-now calendar.selected))}}
+                  {{if hasValidExpiration (concat "Expires " (moment-from-now calendar.selected))}}
```

Comment:
So this will read **Expires X days** instead of **Expires in X days**, I think the latter is grammatically correct?

Reply:
It was actually reading "in" twice so I decided to remove the hardcoded "in".

<img width="360" alt="screen shot 2018-09-28 at 3 08 10 pm" src="https://user-images.githubusercontent.com/16459843/46235685-5a8f7e00-c330-11e8-8483-c280c12e7f16.png">